charts/em-stitching/values.yaml

```diff
@@ -58,3 +58,6 @@ java:
     SERVER_PORT: 8080
     TASK_ENV: documentTaskLock-{{ .Values.global.environment }}
     CALLBACK_MAX_ATTEMPTS: 3
+    HISTORIC_DOCUMENT_TASKS_NO_OF_DAYS: 180
+    HISTORIC_DOCUMENT_TASKS_NO_OF_RECORDS: 1000
+    HISTORIC_DOCUMENT_TASKS__RETENTION_MILLISECONDS: 0 */30 * * * ?
```

Comment:
this looks like a schedule, not a retention period

Reply:
Will update the var name.